py/desimodel/install.py

```diff
@@ -84,7 +93,8 @@ def install(desimodel=None, version=None):
     proc = Popen(command, stdout=PIPE, stderr=PIPE)
     out, err = proc.communicate()
     status = proc.returncode
-    return status
```

Comment:
Because it would always be zero since any non-zero return code is converted into a RuntimeError wrapping the subprocess stderr message.